app/assets/src/components/Login.jsx

```diff
@@ -66,23 +69,37 @@ class Login extends React.Component {
   }
 
   isFormInValid() {
-    if (this.refs.email.value === "" && this.refs.password.value === "") {
-      this.setState({
-        showFailedLogin: true,
-        errorMessage: "Please enter email and password"
+    const logError = () =>
```

Comment:
do you want to add a call a few lines above this one to `errorMessage: "Invalid Email and Password"`?

Reply:
good catch